tests/VolleyManagement.UnitTests/Services/GameService/ExpectedExceptionMessages.cs

```diff
@@ -43,5 +43,7 @@ public static class ExpectedExceptionMessages
         + " the points difference must be equal to 2.";
 
         public const string GAME_DATE_NOT_SET = "Game date not set.";
+
+        public const string SAME_TEAM_IN_ROUND = "Team  has game scheduled already in this round";
```

Comment:
2 spaces between words

Reply:
because the tested method throws an argument exception "Team {Team.Name} has game scheduled already in this round". In the TestMethod I am creating a game with a home team without name (just ID). So, it's why the tested method returns that message without name and with two spaces: before and after name which is empty.

Reply:
Is it hard to give a name?